roles/config-rh-sso/tasks/realm.json

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
See comment above re: .json file (including making a jinja2 template if we need to keep it)